### PR TITLE
Create OS1 issue template

### PR DESCRIPTION
**Pull request by vvisigoth**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/os1-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (matildepark) [inline .github/ISSUE_TEMPLATE/os1-bug-report.md]: Usually we add "actual behaviour" too

Comment (vvisigoth) [reply .github/ISSUE_TEMPLATE/os1-bug-report.md]: I felt like this was covered under repro.

File: .github/ISSUE_TEMPLATE/os1-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+**Screenshots**
+If applicable, add screenshots to help explain your problem. If possible, please also screenshot your browser's dev console. Here are [Chrome's docs](https://developers.google.com/web/tools/chrome-devtools/open) for using this feature.
```
Comment (Fang-) [inline .github/ISSUE_TEMPLATE/os1-bug-report.md]: Do I smell browser bias? Should we maybe include instructions for the less-tested browsers? (;

File: .github/ISSUE_TEMPLATE/os1-bug-report.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (Fang-) [inline .github/ISSUE_TEMPLATE/os1-bug-report.md]:
```suggestion
 - OS: [e.g. MacOS Catalina]
```

**Comment by matildepark** [comment]
This should probably get merged now that we're live.

**Pull request merged**